docs/modules/general/pages/local-env.adoc

```diff
@@ -0,0 +1,26 @@
+= OneCX Local Environment
+
+== Overview
+With OneCX platform, every Application needs to adjusted so it can be integrated within the platform. For the best experience with Application development and integration, OneCX offers a https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository. It provides a consistent setup of the platform and Core Applications, with minimal actions and allows extention.
+
+== Setting Up OneCX and Core Applications
+Using https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository allows to:
+
+* set up all required services to run the OneCX platform
+* set up Shell and other Core Applications with preconfigured data
+* develop and test Applications already integrated with the platform
+* easily extend and overwrite the configuration of the platform and Applications
+
+=== Prerequisites
+It is recommended to get familiar with the README of the https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository to make sure the machine's configuration meets the expectations. It is important to follow it cautiously. Assuming all the steps have been completed, it should be possible to access the OneCX platform locally with preconfigured Shell and Core Applications.
+
+=== Local Development
+// TODO: Link to standalone
+// TODO: Link to sharing packages, webpack?
+While it is possible to develop Applications in an isolation or in a standalone mode, it is recommended to always develop using the https://github.com/onecx/onecx-local-env[OneCX Local Enviornment]. It is the closest configuration to the one in production, that utilizes Webpack configuration, Microfrontends approach and does not require any additional setup to ensure correct integration with OneCX platform.
+
+https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository offers several approaches on how to setup Applications for local development allowing different possibilities.
+
+=== Environment Extending
+// TODO: This part is not documented well on local-env
+https://github.com/onecx/onecx-local-env[OneCX Local Enviornment] repository already takes care of the base configuration for the Shell and Core Applications. By modifying contents of `imports` and `init-data` directories, it is possible to tune the platform by including an additional data for Core Applications or adding data for new ones. It is especially useful for keeping the platform consistent for every team member and reverting the platform to the default state in any time.
```

Comment:
**The** OneCX Local Enviornment ...
... for every team member and reverting the platform to the default state ~~in~~ **at** any time.